armory/data/librispeech/librispeech_dev_clean_split.py

```diff
@@ -207,5 +251,6 @@ def _generate_librispeech_examples(directory):
                     "chapter_id": chapter_id,
                     "speech": os.path.join(path, audio_file),
                     "text": transcript,
+                    "label": speaker_id,
```

Comment:
I think this needs to be str(speaker_id) so that it can be matched with the class _LABEL map

Reply:
I've made the change, and now I'm going to test it on galileo

Reply:
Testing on galileo went fine.